cmd/watch-termination/main.go

```diff
@@ -53,6 +65,54 @@ func main() {
 		klog.SetOutputBySeverity("INFO", stderr)
 	}
 
+	var client kubernetes.Interface
+	if len(*kubeconfigPath) > 0 {
```

Comment:
check for nil ?

Reply:
no. It can never be nil by definition of the flag library.

Reply:
if it is a pointer it can be nil :-) I don't trust godoc.

Reply:
It is the contract from the golang standard library. It's only a pointer because it is a reference to the actual value of that flag. Unfortunately, Golang has no concept of a non-nil pointer (i.e. a reference).